src/server/services/focus/focus-toxx-helpers.ts

```diff
@@ -87,10 +87,7 @@ function getDocumentStepNotificationLink(
   document: FocusDocument,
   stepLabels: FocusStepContent
 ): LinkProps | undefined {
-  if (stepLabels.notification?.link) {
-    return stepLabels.notification?.link(document);
-  }
-  return undefined;
+  return stepLabels.notification?.link?.(document);
```

Comment:
This is not the correct syntax. It's sufficient to do: `return stepLabels.notification?.link(document);` If link isn't found on notification it won't do anything.

Reply:
This gives a typescript error because cannot invoke a function when undefined. In typescript this is actually accepted syntax afaik.

Reply:
You are correct. I've misinterpreted the syntax.